docs/guide/configuration.md

```diff
@@ -129,19 +133,17 @@ Check out [@ts-safeql-demos/multi-connections](https://github.com/ts-safeql/safe
 }
 ```
 
-### What is "connectionUrl" and should I supply it?
+### What is `connectionUrl` and should I configure it?
 
 ::: info TL;DR
-If your'e using migrations and your postgres database URL is different than
+If you're using migrations and your PostgreSQL superuser credentials are different
+than the default below, you will need to configure `connectionUrl`.
```

Comment:
The role doesn't have to be superuser as long as it's able to run `createdb` and `dropdb`. Maybe we should clarify that.

Reply:
Ok, I added 3bf6c0472aa6c9d612ec79eb0579130f140ed2bd for this

I made it a separate paragraph as an "exceptional" case because I'm guessing that many configurations will be using a superuser role

What do you think?

Reply:
Looks great!